src/index.js

```diff
@@ -20,12 +29,74 @@ const positionSeven = board.grid[6];
 const positionEight = board.grid[7];
 const positionNine = board.grid[8];
 
-document.getElementById('one').innerHTML = positionOne;
-document.getElementById('two').innerHTML = positionTwo;
-document.getElementById('three').innerHTML = positionThree;
-document.getElementById('four').innerHTML = positionFour;
-document.getElementById('five').innerHTML = positionFive;
-document.getElementById('six').innerHTML = positionSix;
-document.getElementById('seven').innerHTML = positionSeven;
-document.getElementById('eight').innerHTML = positionEight;
-document.getElementById('nine').innerHTML = positionNine;
+const currentPlayer = () => {
+  whosPlaying.innerHTML = `Player ${game.currentPlayer} moves`
+}
+
+const showBoard = () => {
+  one.innerHTML = positionOne;
+  two.innerHTML = positionTwo;
+  three.innerHTML = positionThree;
+  four.innerHTML = positionFour;
+  five.innerHTML = positionFive;
+  six.innerHTML = positionSix;
+  seven.innerHTML = positionSeven;
+  eight.innerHTML = positionEight;
+  nine.innerHTML = positionNine;
+  currentPlayer()
+}
+
+
+const startGame = () => {
+  showBoard();
+}
+
+startButton.addEventListener("click", startGame)
+
+one.addEventListener("click", function(){
+  gameController.takeTurn(1);
+  one.innerHTML = board.grid[0];
+  gameController.togglePlayer()
+  currentPlayer();
+  ;
+});
+
+two.addEventListener("click", function(){
```

Comment:
These are all pretty similar; we should be able to remove some duplication here.
If we add the event listener to `player-grid`, we should be able to handle any clicks anywhere on the grid. Then we'll be able to see which cell was clicked with the `event.target` 

ie (pseudo code, so might not work)

```
document.getElementById('player-grid').addEventListener('click', singleTurn)

function singleTurn(event) {
  const cellIdWord = event.target.id;
  const cellIdNumber = numbers[cellIdWord];
  gameController.takeTurn(cellIdNumber);
  document.getElementById(cellIdWord) = board.grid[cellIdNumber - 1];
}

const numbers = {
  'one': 1,
  'two': 2,
...
  'nine': 9
}

```
Once this is working, we could then consider changing the cell IDs to numbers (ie 'one' become '1', to avoid needing to convert words to numbers) 

Happy to work through this together if it's not clear!

Reply:
Thanks Dan, I'll give this a go 👍